### DISCUSSION TITLE
when should "derived" be used?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#597] [Q&A]
We are seeing "derived" being used as a negotiated_type with a negotiation_arrangement of "ffs".

Shouldn't "derived" only be used with a "negotiation_arrangement of "capitation"?

**Comment by dgolden** [comment]
I have a tip to share, based on a conversation today with a BCBS company. Their MRF are available both on their company web site _as well as_ on the BCBS association site which (I'm told) has MRF for all 35 (?) BCBS companies .. but the MRFs are different.  The MRF posted on their company web site includes managed care plans. In contrast, the managed care plans are excluded from the MRF sent to the association, since managed care pricing data is irrelevant to subscribers belonging to other companies.
